tests/test_utils.sh

```diff
@@ -694,6 +694,73 @@ function test_path_format_last_dir() {
   assertEquals "full directory formatting with multichar separator" "{l}pathname" "$lp_path_format"
 }
 
+function test_lp_fill {
+    local lp_fill
+
+    COLUMNS=80
+    _lp_fill "Left" "Right" " "
+    assertEquals "full width" 80 ${#lp_fill}
+
+    COLUMNS=3
+    _lp_fill "L" "R" "-"
+    assertEquals "single fill" "L-R" "$lp_fill"
+
+    _lp_fill "L" "R" ""
+    assertEquals "defaults to space" "L R" "$lp_fill"
+
+    COLUMNS=5
+    _lp_fill "L" "R" "~"
+    assertEquals "simple fill 5" "L~~~R" "$lp_fill"
+
+    COLUMNS=6
+    _lp_fill "L" "R" "+-"
+    assertEquals "multi-fill 6" "L+-+-R" "$lp_fill"
+
+    _lp_fill "L" "R" "123" 1
+    assertEquals "multi-fill 6 split" "L1231R" "$lp_fill"
+
+    _lp_fill "L" "R" "${_LP_OPEN_ESC}${_LP_CLOSE_ESC}123" 0
+    __lp_strip_escapes "$lp_fill"
+    # FIXME assertEquals does not see escaped stuff.
+    # assertEquals "multi-fill 6 with escape and no split" "L${_LP_OPEN_ESC}${_LP_CLOSE_ESC}123 R" "$ret"
+    assertEquals "multi-fill 6 with escape and no split" "L123 R" "$ret"
+
+    _lp_fill "L" "R" "${_LP_OPEN_ESC}${_LP_CLOSE_ESC}123" 1
+    __lp_strip_escapes "$lp_fill"
+    # assertEquals "multi-fill 6 with escape and split" "L${_LP_OPEN_ESC}${_LP_CLOSE_ESC}1231R" "$ret"
+    assertEquals "multi-fill 6 with escape and split" "L1231R" "$ret"
+
+    COLUMNS=11
+    _lp_fill "Left" "Right" "="
+    assertEquals "regular fill 11" "Left==Right" "$lp_fill"
+
+    _lp_fill "Le" "Ri" "+-"
+    assertEquals "multi-fill 11 split default" "Le+-+-+-+Ri" "$lp_fill"
+
+    _lp_fill "Le" "Ri" "+-" 0
+    assertEquals "multi-fill 11 no split" "Le+-+-+- Ri" "$lp_fill"
+
+    _lp_fill "Le" "Ri" "+-" 1
+    assertEquals "multi-fill 11 explicit split" "Le+-+-+-+Ri" "$lp_fill"
+
+    _lp_fill "Le" "Ri" "123" 1
+    assertEquals "multi-fill 11 with split" "Le1231231Ri" "$lp_fill"
+
+    _lp_fill "Le" "Ri" "${_LP_OPEN_ESC}${_LP_CLOSE_ESC}123" 0
+    __lp_strip_escapes "$lp_fill"
+    # assertEquals "multi-fill 11 with escape and no split" "Le${_LP_OPEN_ESC}${_LP_CLOSE_ESC}123123 Ri" "$ret"
+    assertEquals "multi-fill 11 with escape and no split" "Le123123 Ri" "$ret"
+
+    _lp_fill "Le" "Ri" "${_LP_OPEN_ESC}${_LP_CLOSE_ESC}123" 1
+    __lp_strip_escapes "$lp_fill"
+    # assertEquals "multi-fill 11 with escape and split" "Le${_LP_OPEN_ESC}${_LP_CLOSE_ESC}1231231Ri" "$ret"
+    assertEquals "multi-fill 11 with escape and split" "Le1231231Ri" "$ret"
+
+    COLUMNS=32
+    _lp_fill "Left part·" "·right part" "⣀⠔⠉⠢" 1
+    assertEquals "beautiful fill" "Left part·⣀⠔⠉⠢⣀⠔⠉⠢⣀⠔⠉·right part" "$lp_fill"
```

Comment:
This test (and only this test) is failing on Windows. I am not sure why...

Reply:
FTR:
```
shunit2:ERROR test_lp_fill() returned non-zero return code.
ASSERT:beautiful fill expected:<Left part·⣀⠔⠉⠢⣀⠔⠉⠢⣀⠔⠉·right part> but was:<Left part·⣀⠔⠉·right part>
```

I can't easily debug, not having Windows at hand, but this is strange indeed.

Reply:
I do. I have never run the tests locally on Windows before, I always let Github Actions run them. Wow, they are so slow.

That test passes for me. This makes me think it is an encoding issue on the runner. I can't find any docs that suggest that the encoding of the Windows image is not UTF-8, but I don't know what else it would be.

I see a few possible solutions:
* Try to force bash to UTF-8 encoding.
* Try to force the runner image to UTF-8 encoding.
* Give up and tell the tests to skip this test if it is on a Windows VM. Put this block before the test:
```bash
# The Windows runners have issues with these Unicode characters.
if [[ ${RUNNER_OS-} == "Windows" ]]; then
  startSkipping
fi
```

Reply:
I let you decide, I have absolutely no clue, and I did not touched such things under Windows for literally decades.

Reply:
Go for the last option, it's not worth the headache.

I would add those 4 lines on line 749.